src/cookies.ts

```diff
@@ -61,8 +65,8 @@ export const setCookie = ({
 }): void => {
 	const expires = new Date();
 
-	if (!isValidCookieValue(name) || !isValidCookieValue(value)) {
-		return;
+	if (!isValidCookie(name, value)) {
```

Comment:
do you think it would be worth abstracting these two check into a `validateCookie()` function that throws, rather than having two instances of the same behaviour?

i'm not sure. just a suggestion – whichever you're happy with is good with me?

Reply:
or maybe `throwIfInvalid()` might be more explicit

Reply:
Good point! Personally I like that it's explicit that the function will not continue if the cookie is invalid. But I can see the argument to consolidate.

However overriding that is your suggestion in the previous PR, to merge `setCookie` and `setSessionCookie` so there would be only one `throw` in future anyway. I will do another PR for that.